src/core/Plugin.php

```diff
@@ -1675,13 +1675,30 @@ public function filter_the_content($content)
         // Check if it is configured to append to the content
         $append_to_content = 'yes' === $legacyPlugin->modules->multiple_authors->options->append_to_content;
 
-        if ($this->should_display_author_box() && $append_to_content) {
+        if (!$this->ppma_post_author_box_disabled() && $this->should_display_author_box() && $append_to_content) {
```

Comment:
@ojopaul Maybe you could move the method the condition `!$this->ppma_post_author_box_disabled()` to inside the method `should_display_author_box`? So everywhere it is used, that option will be checked?

Reply:
This has been updated @andergmartins